### PR TITLE
App structure update

### PR DESCRIPTION
**Pull request by adghayes**
Updates `vxsuite-complete-system` to build and run correctly after the following changes in the directory structure of `vxsuite`:
- https://github.com/votingworks/vxsuite/pull/2997
- https://github.com/votingworks/vxsuite/pull/2996

### CODE REVIEW

**Review by carolinemodic — CHANGES_REQUESTED** [review]
You'll need to either change this line and the one below back to `vx-${APP}` or change these filenames: https://github.com/votingworks/vxsuite-complete-system/blob/main/setup-machine.sh#L365

**Comment by adghayes** [comment]
> You'll need to either change this line and the one below back to `vx-${APP}` or change these filenames: https://github.com/votingworks/vxsuite-complete-system/blob/main/setup-machine.sh#L365

@carolinemodic are you talking about the `config/${APP}.service` files? I changed those, at least.

File: vxdev/update-code.sh [hunk 1]
```diff
@@ -74,24 +74,24 @@ cp /vx/config/.env.local vxsuite/.env.local
 make build-kiosk-browser
 echo $APP_TYPE
 if [[ $APP_TYPE == 'VxCentralScan' ]] || [[ $APP_TYPE == 'VxAdminCentralScan' ]]; then
-	cp /vx/config/.env.local vxsuite/services/scan/.env.local
-	cp /vx/config/.env.local vxsuite/frontends/bsd/.env.local
+	cp /vx/config/.env.local vxsuite/apps/central-scan/backend/.env.local
+	cp /vx/config/.env.local vxsuite/apps/central-scan/frontend/.env.local
 	./build.sh bsd
```
Comment (amcmanus) [inline vxdev/update-code.sh:79]: I think this should be `./build.sh central-scan`

**Review by amcmanus — CHANGES_REQUESTED** [review]

**Comment by carolinemodic** [comment]
> > You'll need to either change this line and the one below back to `vx-${APP}` or change these filenames: https://github.com/votingworks/vxsuite-complete-system/blob/main/setup-machine.sh#L365
> 
> @carolinemodic are you talking about the `config/${APP}.service` files? I changed those, at least.

Oh my bad, missed that in my review last night

**Comment by carolinemodic** [comment]
Not sure what we tested on this but I would recommend testing the build/run scripts and then setup-machine for at least one app here, I can test the VxDev stuff after you merge

**Comment by adghayes** [comment]
Thanks @amcmanus for the catch.

@carolinemodic will test those scripts.

I know @amcmanus is moving things around so this may be moot, but is there any reason we can't have a CircleCI check that simply tries to run the `build` and `run` scripts and confirms they exist successfully? I know that's probably not possible with lockdown, and maybe challenging with `setup-machine`, but for `build` and `run` it sounds doable.

**Comment by amcmanus** [comment]
> Thanks @amcmanus for the catch.
> 
> @carolinemodic will test those scripts.
> 
> I know @amcmanus is moving things around so this may be moot, but is there any reason we can't have a CircleCI check that simply tries to run the `build` and `run` scripts and confirms they exit successfully? I know that's probably not possible with lockdown, and maybe challenging with `setup-machine`, but for `build` and `run` it sounds doable.

@adghayes, eventually, I'd love to have those steps automated in CircleCI or similar. The trick is being sure that whatever image/container our CircleCI configs use is an accurate reflection of our production release environment. That said, the Packer work I'm going to demo today will make for a useful sanity check until we get further along with automated testing of our builds. It's how I've caught a few build issues so far. Looking forward to your feedback on it.

**Comment by adghayes** [comment]
@carolinemodic the `build.sh` and `run.sh` scripts on all apps and ran the `setup-machine.sh` script on `admin`, all works!

**Review by carolinemodic — APPROVED** [review]

**Review by amcmanus — APPROVED** [review]

**Pull request merged**